i18n/utils.py

```diff
@@ -25,10 +25,6 @@ class I18nFileWrapper:
     def i18n_dir(cls):
         return os.path.dirname(__file__)
```

Comment:
I believe this should be something like

```suggestion
        return settings.I18N_DIR
```